packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -493,7 +493,7 @@ export class PeerManager {
   private async requestPing(peer: PeerId): Promise<void> {
     const peerIdStr = peer.toString();
     try {
-      this.logger.warn("requestPing", {peer: peer.toString()});
+      this.logger.debug("requestPing", {peer: peer.toString()});
```

Comment:
same here, consider completely removing this log, seems like an artifact of very early debugging?